cfgov/diversity_inclusion/jinja2/diversity_inclusion/voluntary-assessment-form.html

```diff
@@ -12,132 +12,217 @@
+                                   for="{{ form.tax_id.id_for_label }}">
+                                {{ form.tax_id.label }}
+                            </label>
+                            {{ form.tax_id }}
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <h2>Contact information</h2>
+            <p>
+                Please provide the name and contact information for the individual who will be your institution’s POC.
+            </p>
+            <div class="o-well u-mb30">
+                <div class="o-form_group">
+                    <div class="m-form-field">
+                        <label class="a-label a-label__heading"
+                               for="{{ form.contact_name.id_for_label }}">
+                            {{ form.contact_name.label }}
+                        </label>
+                        {{ form.contact_name }}
+                    </div>
+                </div>
+                <div class="o-form_group">
+                    <div class="m-form-field">
+                        <label class="a-label a-label__heading"
+                               for="{{ form.contact_title.id_for_label }}">
+                            {{ form.contact_title.label }}
+                        </label>
+                        {{ form.contact_title }}
+                    </div>
+                </div>
+                <div class="o-form_group">
+                    <div class="m-form-field">
+                        <label class="a-label a-label__heading"
+                               for="{{ form.contact_email.id_for_label }}">
+                            {{ form.contact_email.label }}
+                        </label>
+                        {{ form.contact_email }}
+                    </div>
+                </div>
+                <div class="o-form_group">
+                    <div class="m-form-field">
+                        <label class="a-label a-label__heading"
+                               for="{{ form.contact_phone.id_for_label }}">
+                            {{ form.contact_phone.label }}
+                        </label>
+                        {{ form.contact_phone }}
+                    </div>
+                </div>
+                <div class="o-form_group">
+                    <div class="m-form-field">
+                        <label class="a-label a-label__heading"
+                               for="{{ form.contact_phone_alt.id_for_label }}">
+                            {{ form.contact_phone_alt.label }}
+                            <small class="a-label_helper">(optional)</small>
+                        </label>
+                        {{ form.contact_phone_alt }}
+                    </div>
+                </div>
+            </div>
+            <div class="o-form_group">
+                <input type="hidden"
+                       name="csrfmiddlewaretoken"
+                       value="{{ csrf_token }}">
```

Comment:
Trivial: These attributes are short enough to leave on line line, IMO.

Reply:
Fixed in 612d440